docs/release-notes/flink-1.11.md

```diff
@@ -0,0 +1,220 @@
+
+### PyFlink
+#### Throw exceptions for the unsupported data types ([FLINK-16606](https://issues.apache.org/jira/browse/FLINK-16606))
+DataTypes can be configured with some parameters, e.g., precision.
+However, the precision provided by users takes no effect but a default value will always be used. 
+To avoid confusion since Flink 1.11 exceptions will be thrown if the value is not supported to make it more visible to users. 
+Changes include:
+- the precision for `TimeType` could only be `0`
+- the length for `VarBinaryType`/`VarCharType` could only be `0x7fffffff`
+- the precision/scale for `DecimalType` could only be `38`/`18`
+- the precision for `TimestampType`/`LocalZonedTimestampType` could only be `3`
+- the resolution for `DayTimeIntervalType` could only be `SECOND` and the `fractionalPrecision` could only be `3`
+- the resolution for `YearMonthIntervalType` could only be `MONTH` and the `yearPrecision` could only be `2`
+- the `CharType`/`BinaryType`/`ZonedTimestampType` is not supported
+
+### Monitoring
+#### Bla ([FLINK-](https://issues.apache.org/jira/browse/FLINK-))
+
+#### Converted all MetricReporters to plugins ([FLINK-16963](https://issues.apache.org/jira/browse/FLINK-16963))
+All MetricReporters that come with Flink have been converted to plugins.
+They should no longer be placed into `/lib` directory (doing so may result in dependency conflicts!), but `/plugins/<some_directory>` instead.
+
+#### Changed of DataDog's metric reporter Counter metrics ([FLINK-15438](https://issues.apache.org/jira/browse/FLINK-15438))
+The DataDog metrics reporter now reports counts as the number of events over the reporting interval, instead of the total count. 
+This aligns the count semantics with the DataDog documentation.
+
+#### Switch to Log4j 2 by default ([FLINK-15672](https://issues.apache.org/jira/browse/FLINK-15672))
+Flink now uses Log4j2 by default. 
+Users who wish to revert back to Log4j1 can find instructions to do so in the logging documentation.
+
+#### Changed behaviour of JobManager API's log request ([FLINK-16303](https://issues.apache.org/jira/browse/FLINK-16303))
+Requesting an unavailable log or stdout file from the JobManager's HTTP server returns status code 404 now. 
+In previous releases, the HTTP server would return a file with `(file unavailable)` as its content.
+
+#### Removal of lastCheckpointAlignmentBuffered metric ([FLINK-16404](https://issues.apache.org/jira/browse/FLINK-16404))
+Note that the metric of `lastCheckpointAlignmentBuffered` has been removed, because the upstream task will not send any following data after barrier until alignment on downstream side. 
+But this info in web UI still exists and always shows 0 now. 
+We will also remove it from UI in a follow-up separate ticket future.
+
+### Connectors
+#### Dropped Kafka 0.8/0.9 connectors ([FLINK-15115](https://issues.apache.org/jira/browse/FLINK-15115))
+The Kafka 0.8 and 0.9 connectors are no longer under active development. 
```

Comment:
I think you cannot use Kafka connectors compiled against an older Flink version. This is mentioned in the release notes of https://issues.apache.org/jira/browse/FLINK-17376, which is not yet added here.

Reply:
@zentol can you clarify this? This comes directly from https://issues.apache.org/jira/browse/FLINK-15115

Reply:
I assumed that the connectors work against non-deprecated @Public APIs, which according to our contracts are supposed to work until 2.0.0 . If that is not the case, then we have to adjust the release notes naturally.

Reply:
I remove the second half of the sentence from https://issues.apache.org/jira/browse/FLINK-15115. We should update also the text here.

Side note, I remember two or three occurrences of this by now, where a `@Public` class uses internally APIs that are not `@Public`. We're currently not verifying this. We would have to transitively check if any "public" code uses purely public code. I can start a ML discussion about this, at least to bring some awareness.